### PR TITLE
Create User in admin

### PR DESCRIPTION
**Pull request by mariefontanel**
You should know that we have a default password (str_random) because of the not null value in the database.
We will have to modify it later when there will be the automatic generation of the password of a user when it will be registered directly in the BO.

### CODE REVIEW

File: app/Http/Controllers/Admin/UserController.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $this->validate($request,[
+            'gender' => 'required|integer',
+            'lastname' => 'required|string|max:45',
+            'firstname' => 'required|string|max:45',
+            'birthdate' => 'required|date',
+            'password' => 'string|min:6|max:191|confirmed|nullable',
```
[the remark this answers is not in the document]
Comment (RMaxence) [reply app/Http/Controllers/Admin/UserController.php]: problem:

lastname = 2
firstname = 1
birthdate = 2028-08-11 (futur)
adress 1 = +
zip code = +
city = 123456

everything got validated

**Comment by mariefontanel** [comment]
We have to talk together about this !

**Comment by Noksit** [comment]
Peut etre serait-ce mieux de mettre
route '/users/create' au lieu de '/users-create' afin d'etre comme tout le monde

**Comment by mariefontanel** [comment]
For the routes, we have to make the point to do all the same but actually the official doc is like you!

**Pull request merged**